gazebo_plugins/src/gazebo_ros_hand_of_god.cpp

```diff
@@ -137,15 +140,25 @@ void GazeboRosHandOfGod::Load(gazebo::physics::ModelPtr _model, sdf::ElementPtr
 
 void GazeboRosHandOfGodPrivate::OnUpdate()
 {
+#ifdef IGN_PROFILER_ENABLE
+  IGN_PROFILE("GazeboRosHandOfGodPrivate::OnUpdate");
+  IGN_PROFILE_BEGIN("lookupTransform");
+#endif
   // Get TF transform relative to the /world frame
   geometry_msgs::msg::TransformStamped hog_desired_tform;
   try {
     hog_desired_tform = buffer_->lookupTransform("world", frame_ + "_desired", tf2::TimePointZero);
   } catch (const tf2::TransformException & ex) {
     RCLCPP_WARN_ONCE(ros_node_->get_logger(), "%s", ex.what());
+#ifdef IGN_PROFILER_ENABLE
+    IGN_PROFILE_END();
+#endif
     return;
   }
-
+#ifdef IGN_PROFILER_ENABLE
+  IGN_PROFILE_END();
```

Comment:
is there a matching `IGN_PROFILE_BEGIN` for this `IGN_PROFILE_END`?

Reply:
This one 

https://github.com/ros-simulation/gazebo_ros_pkgs/blob/593a878ffe1746574fd31777cfd7c9996ee2b52e/gazebo_plugins/src/gazebo_ros_hand_of_god.cpp#L145

The other `IGN_PROFILE_END` it's in case of return.